pyuvsim/uvsim.py

```diff
@@ -366,16 +297,7 @@ def run_uvdata_uvsim(input_uv, beam_list, beam_dict=None, catalog=None, source_l
             {antenna_name : beam_id}, where beam_id is an index in
                    the beam_list. This is used to assign beams to antennas.
                    Default: All antennas get the 0th beam in the beam_list.
-        source_list_name: string
-            Name of the catalog
-        catalog: np.ndarray in shared memory
-            Immutable source parameters
-        obs_param_file: string
-            Parameter filename if running from config files.
-        telescope_config_file: string
-            Telescope configuration file if running from config files.
-        antenna_location_file: string
-            antenna_location file if running from config files.
+        catalog: array of source.Source objects
```

Comment:
This is incorrect. The catalog is passed in as an ndarray in shared memory (see earlier version of the docstring). The Source class no longer exists.

Reply:
I reverted to the previous docstring.